modules/file-integrity-operator-installing-cli.adoc

```diff
@@ -27,8 +27,10 @@ kind: Namespace
 metadata:
   labels:
     openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged <1>
```

Comment:
@sheriff-rh Actually I just realized... the PSA enforcement that was scheduled for 4.12 is getting pushed to 4.13 now. So I'm wondering if this change should actually not go in for 4.12 anymore...

@stlaz @ibihim With the restricted enforcement for PSA getting rolled back/moving to 4.13, should we be updating docs with these kinds of changes, or not? I'm aware of at least one other PR (#51481) that has added this label, and need to know if we should revert that and communicate out that we shouldn't make these changes yet for 4.12. Or if it's okay to add for 4.12?

Reply:
This kind of documentation is fine to be added even for lower versions. Properly setting namespace labels is a good thing irregardless of whether we are globally enforcing PSa or not.